programs/clearing_house/src/controller/orders.rs

```diff
@@ -417,6 +429,14 @@ pub fn fill_order(
         "Order must be triggered first"
     )?;
 
+    {
+        let market = market_map.get_ref(&market_index)?;
+        validate!(
+            market.current_liquidations == 0 || user.is_being_liquidated(),
```

Comment:
should this be `market.current_liquidations != 0` ? So if there are liquidations going on in a market, orders can't be filled?

Reply:
This check makes it so if there are on going liquidations, only the user being liquidated can be filled (though this is going to change)